NSURLSessionCancellation/ViewController.swift

```diff
@@ -12,69 +12,69 @@ class ViewController: UIViewController {
 
     // A sample URL that takes a while to download (so that cancellaton can be demonstrated)
     
-    private let url = NSURL(string: "http://yahoo.com")!
+    fileprivate let url = URL(string: "http://yahoo.com")!
```

Comment:
Just leave it then. I was just wondering. Usually I start restrictive until I need to expose but not sure it's the right thing. Like you said either is fine here.